src/main/scala/modbat/mbt/Main.scala

```diff
@@ -5,7 +5,7 @@ import modbat.config.Version
 import modbat.log.Log
 
 object Main {
-  val config = new Configuration()
+  var config = new Configuration()
```

Comment:
Is this change from `val` to `var` needed?

Reply:
Oops...I'm not sure how it is changed. Maybe a mistake happened when editing, but I will change it back.